jenkins/model_output_processing_Jenkinsfile

```diff
@@ -71,6 +72,7 @@ pipeline {
           aws s3 cp model_output_categorized.csv s3://prod-owi-resources/resources/Application/wbeep/model_output_csv/model_output_categorized_${DATE}.csv
           echo ${DATE} > date.txt
           aws s3 cp date.txt s3://prod-owi-resources/resources/Application/wbeep/${TIER}/date.txt
+          aws s3 cp . s3://wbeep-${TIER}-website/estimated-availability/date/test_results/ --recursive --exclude "*" --include "*${DATE}.csv" --include "order_of_magnitude_test.txt"
```

Comment:
pushes the test results and the order of magnitude log up to each tier's public-facing bucket so that the email can provide links to these files instead of attaching them by default